src/pages/[platform]/deploy-and-host/fullstack-branching/pr-previews/index.mdx

```diff
@@ -44,4 +40,91 @@ With fullstack previews, you can set up ephemeral fullstack environments on ever
+### Prerequisites
+
+Before you get started, make sure you have the following:
+
+- [An Amplify Gen 2 app created](https://docs.amplify.aws/gen2/start/quickstart/)
+- Ensure that your git repository is private. For security purposes, fullstack previews are disabled for public repositories with Amplify backend templates.
+
+
+### Enable fullstack previews
+
+To enable fullstack web previews for your Amplify Gen 2 app, follow these steps:
+
+1. Login to the [Amplify console](https://console.aws.amazon.com/amplify/home) and select your app.
+
+2. Navigate to **Hosting > Previews**. Select the **`main`** branch and click on **Edit settings**. 
+![Amplify console page displaying the list of branches for the previews functionality](/images/gen2/fullstack-branching/previews-1.png)
+
+3. Click on the **Pull request previews** toggle button and choose **Confirm** to enable previews.
+![Amplify Console page displaying a toggle button to enable the previews functionality](/images/gen2/fullstack-branching/previews-2.png)
+
+4. Done! You have successfully enabled previews on the production branch.
+![Amplify Console page displaying the main branch with previews functionality enabled](/images/gen2/fullstack-branching/previews-3.png)
+
+5. Ship updates to the `dev` branch. Now, when you create a pull request for the `main` branch, Amplify will build and deploy your fullstack PR and provide you with a preview URL.
+![Amplify Console page displaying the main, dev, and preview branch](/images/gen2/fullstack-branching/previews-4.png)
+
+For **GitHub repositories only**, you can access your preview URL directly on the pull request from the Amplify Hosting's bot comment:
+
+![GitHub pull request displaying preview URL in a bot comment](/images/gen2/fullstack-branching/previews-5.png)
+
+After the pull request is merged or closed, the preview URL is deleted and any ephemeral fullstack environment is also deleted.
+
+### Share backend resources across Preview branches
+
+Fullstack previews allow teams a way to preview changes from pull requests before merging code to a production branch. Pull requests let you tell others about changes you’ve pushed to a branch in a repository and the changes can be reviewed by accessing the preview URL. When previews are enabled on a git branch, by default every pull request created against the git branch creates an ephemeral fullstack environment.
+
+In some instances, you may not want to deploy new resources for every preview branch. For example, you might want all your preview branches to point to the backend resources deployed by the `dev` branch so you can reuse seed data, users, and groups.
+
+To achieve this, you can update your app build settings to reuse backend resources across your preview branches. In the Amplify console, select your app on the **All apps** page. From the **App overview** page, select **Hosting > Build settings** to view your app's build specification YML file.
```

Comment:
nit: Should we spell this out as `YAML`?

```suggestion
To achieve this, you can update your app build settings to reuse backend resources across your preview branches. In the Amplify console, select your app on the **All apps** page. From the **App overview** page, select **Hosting > Build settings** to view your app's build specification YAML file.
```

Reply:
Done!